### ISSUE TITLE
Invalid data: Xdata length invalid

### ISSUE THREAD

**Issue by IronButterfly** [custom-components/ble_monitor#297]
After one of the latest component updates, I see the following error in the HomeAssistant log:

```
Logger: custom_components.ble_monitor
Source: custom_components/ble_monitor/__init__.py:676
Integration: Passive BLE monitor (documentation)
First occurred: 18:12:19 (5 occurrences)
Last logged: 22:12:28

Invalid data: Xdata length invalid
```
Could you please let me know how this can be fixed?

**Comment by Ernst79** [comment]
If your sensor works fine, there is no need to fix anything. It means that the sensor has received a message with invalid data length. In the next update, this message will be moved to debug level, which means you won’t see it anymore.

**Comment by Ernst79** [comment]
Fixed in 1.1.4
